README.md

```diff
@@ -694,9 +694,10 @@ The following setting is optional and allows you to configure image registry cre
 
 * `settings.container-registry.credentials`: An array of container images registry credential settings. Each element specifies the registry and the credential information for said registry.
 The credential fields map to [containerd's registry credential fields](https://github.com/containerd/containerd/blob/v1.6.0/docs/cri/registry.md#configure-registry-credentials), which in turn map to the fields in `.docker/config.json`.
-It is recommended to programmatically set these settings via `apiclient` through the Bottlerocket control container and/or custom host-containers.
 
-  An example `apiclient` call to set registry credentials for `gcr.io` and `docker.io` looks like this:
+  To avoid storing plaintext credentials in external systems, it is recommended to programmatically apply these settings via apiclient using a [bootstrap](#bootstrap-containers-settings) or [host container](#host-containers-settings).
```

Comment:
```suggestion
  To avoid storing plaintext credentials in external systems, it is recommended to programmatically apply these settings via `apiclient` using a [bootstrap](#bootstrap-containers-settings) or [host container](#host-containers-settings).
```